rust-runtime/aws-smithy-runtime-api/src/lib.rs

```diff
@@ -33,8 +33,7 @@ pub mod client;
 
 pub mod http;
 
-/// Internal builder macros. Not intended to be used outside of the aws-smithy-runtime crates.
-#[doc(hidden)]
+/// Builder macros. Not intended to be used outside of the aws-smithy-runtime crates.
 pub mod macros;
```

Comment:
I think we should expand these macros where they're used and delete them. They're not fully baked enough to be stable.

Reply:
Great call out. Updated in [55f1c75](https://github.com/smithy-lang/smithy-rs/pull/3226/commits/55f1c75559c80281eef9cac17db3f54f253d7cc7).

Reply:
Good catch. Currently this module is used within a single module in `aws-smithy-rust-runtime`. Makes sense to inline the expanded code in that module and delete `macros` at least for now. Done in [3e9bcc6](https://github.com/smithy-lang/smithy-rs/pull/3226/commits/3e9bcc6313907fc8100fcddbb8436f2e901d85f4).